JumpscaleLibs/clients/stellar/StellarClient.py

```diff
@@ -56,14 +57,18 @@ def _init(self, **kwargs):
             kp = Keypair.from_secret(self.secret)
         self.address = kp.public_key
 
-    def _get_horizon_server(self)
+    def _get_horizon_server(self):
         return Server(horizon_url=_HORIZON_NETWORKS[str(self.network)])
 
     def get_balance(self):
         """Gets balance for address
         """
-        response= self._get_horizon_server().accounts().account_id(address).call()
-        return (response['balances'])
+        response= self._get_horizon_server().accounts().account_id(self.address).call()
+        balances=j.baseclasses.dict()
```

Comment:
any specific reason to use this one?

Reply:
No, which one should we use?

Reply:
It had a __str and __repr__ implementation  but ignore it for now, working on a specific balances object  that differentiates between lovked, free and multisig balances and has the proper representation methods